### PR TITLE
[HDInsight] Update hdinsight version related to support ssh-public-key parameter

### PR DESCRIPTION
**Pull request by aim-for-better**
This PR is in order to update azure-hdinsight version which is related to previsous PR[ [HDInsight] support ssh_public_key ](https://github.com/Azure/azure-cli/pull/9512)
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by tjprescott** [comment]
Is there an open issue this is addressing?

File: src/command_modules/azure-cli-hdinsight/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+0.3.5
++++++
+* Support --ssh-public-key parameter. Minor fixes.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-hdinsight/HISTORY.rst]: You can remove the "Minor fixes" part. You should be more specific about which commands support `--ssh-public-key`

Comment (aim-for-better) [reply src/command_modules/azure-cli-hdinsight/HISTORY.rst]: > You can remove the "Minor fixes" part. You should be more specific about which commands support `--ssh-public-key`

Thanks for your suggestion. I have updated the history.rst file.

**Comment by aim-for-better** [comment]
> Is there an open issue this is addressing?

This is addressing previous PR  [[HDInsight] support ssh_public_key](https://github.com/Azure/azure-cli/pull/9512)

**Pull request merged**